syndesis/src/modules/dashboard/pages/DashboardPage.tsx

```diff
@@ -132,7 +132,7 @@ export default () => (
                       linkToIntegrations={resolvers.integrations.list()}
                       linkToIntegrationCreation={resolvers.integrations.create.start.selectConnection()}
                       linkToConnections={'/connections'}
```

Comment:
Should you use `resolvers` for this too?

Reply:
I should indeed 😂